examples/run-examples.sh

```diff
@@ -1,36 +1,148 @@
 #!/bin/bash
 # Testscript running all examples from example-list.txt
 #
-# Copyright (c) 2017      Christoph Niethammer <niethammer@hlrs.de>
+# Copyright (c) 2017-2024 Christoph Niethammer <niethammer@hlrs.de>
 #
 
-MARDYN_EXE="mpirun -np 1 $PWD/../src/MarDyn"
-MARDYN_OPTIONS="--steps 10 -v"
+# Some default values
+MARDYN_EXE="$PWD/../src/MarDyn"
+MARDYN_ARGS="--steps 20 -v"
+MPIRUN_EXE="mpirun"
+MPIRUN_ARGS="-n 4 --oversubscribe"
 EXAMPLE_LIST_FILE=${EXAMPLE_LIST_FILE:=example-list.txt}
 LOGFILE=${LOGFILE:=$PWD/run-examples.log}
```

Comment:
Why are you not using the following?
```suggestion
EXAMPLE_LIST_FILE="example-list.txt"
LOGFILE="$PWD/run-examples.log"
```